public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -110,65 +98,48 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         return (true, abi.encode(values, extraData));
     }
 
-    function performUpkeep(bytes calldata performData) external override {
+    // function will be performed on-chain
+    function performUpkeep(bytes calldata performData) external {
+        // Decode incoming performData
         (bytes[] memory signedReports, bytes memory extraData) = abi.decode(
             performData,
             (bytes[], bytes)
         );
 
         bytes memory report = signedReports[0];
 
-        bytes memory bundledReport = bundleReport(report);
+        (, bytes memory reportData) = abi.decode(report, (bytes32[3], bytes));
```

Comment:
i think this bytes32[3] type that gets decoded here is the "report context" .... while unused, it may be helpful to give the decoded and unused value a variable, but then comment it out for being unused:
`(bytes32[3] /* reportContextData */, bytes memory reportData)`

Reply:
Done